### ISSUE TITLE
faceforensics dataset download

### ISSUE THREAD

**Issue by xuanhanyu** [NVlabs/few-shot-vid2vid#79]
**How should I set the parameters of the download script when using the faceforensics dataset? This dataset is too big  and contain different type of subdataset. What should I do ?**

_usage: faceforensics_download_v1.py [-h] [-d DATASET_TYPE] [--not_altered]
                                    [--not_original] [--not_mask] [--not_test]
                                    [--not_train] [--not_val] [--sample_only]
                                    output_path_

Downloads FaceForensics public data release.

positional arguments:
  output_path           directory in which to downloadzhege

optional arguments:
  _-h, --help            show this help message and exit
  -d DATASET_TYPE, --dataset_type DATASET_TYPE
                        Enter which dataset you want to download: "raw",
                        "compressed", "selfreenactment_raw",
                        "selfreenactment_compressed", "original_videos"
                        "source_to_target_images" or "selfreenactment_images".
  --not_altered         don't download face2face altered videos
  --not_original        don't download original videos
  --not_mask            don't download face2face mask videos
  --not_test            don't download videos of the test set
  --not_train           don't download videos of the training set
  --not_val             don't download videos of the validation set
  --sample_only         activate this, if you only want to download 5 files per subfolder_

**Comment by zb-zuishuai** [comment]
python download-FaceForensics.py -d selfreenactment_images(choose 1) --sample_only(choose 2)  ./data
